flink-ai-flow/lib/notification_service/notification_service/proto/notification_service.proto

```diff
@@ -208,4 +215,23 @@ message RegisterClientResponse {
     ReturnStatus return_code = 1;
     string return_msg = 2;
     int64 client_id = 3;
+}
+
+message CloseClientRequest {
```

Comment:
`ClientIdRequest` is fine, it could replace `CloseClientRequest` and `isClientExistsRequest`